File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        
        int n1 = nums.size()/2;

        unordered_map<int,int>mp;
        for(int i=0;i<nums.size();i++){
            mp[nums[i]]++;
        }
        int l1=0;
        for(auto it = mp.begin();it!=mp.end();it++){
            if(it->second>=n1){
                l1= it->first;
            }
        }
        return l1;
    }
};

File: Data Structures & Algorithms/duplicate-integer/submission-1.py
class Solution:
    def hasDuplicate(self, nums: List[int]) -> bool:
        
        mp = {}

        for l1 in nums:
            mp[l1] = mp.get(l1,0) + 1

        for l2 in nums:
            if mp[l2] >1:
                return True

        return False